Commbox.json

```diff
@@ -5611,16 +5611,166 @@
+                      "type": "array",
+                      "items": {
+                        "oneOf": [
+                          {
+                            "type": "object",
+                            "properties": {
+                              "recipient": {
+                                "type": "string",
+                                "description": "Recipient number",
+                                "example": 972546663529
+                              },
+                              "id": {
+                                "type": "string",
+                                "example": "gBEGlyVGZjUpAgmyuqAHfxbHmnQ"
+                              },
+                              "status": {
+                                "type": "integer",
+                                "format": "int32",
+                                "example": 200
+                              },
+                              "object_id": {
+                                "type": "integer",
+                                "format": "int64",
+                                "example": 38383744,
+                                "description": "Object ID. This is a commbox generated number (optional)"
+                              }
+                            }
+                          },
+                          {
+                            "type": "object",
+                            "properties": {
+                              "recipient": {
+                                "type": "string",
+                                "description": "Recipient number",
+                                "example": 972546663529
+                              },
+                              "status": {
+                                "type": "integer",
+                                "format": "int32",
+                                "example": 400
+                              },
+                              "failed": {
+                                "type": "object",
+                                "example": "invalid — Not a valid WhatsApp user"
+                              }
+                            }
+                          }
+                        ]
+                      }
                     }
                   }
                 }
               }
             }
           },
           "400": {
-            "description": "Bad Request - The request could not be understood by the server. Incoming parameters might not be valid"
+            "description": "Provider returned failure",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "example": 400
+                    },
+                    "description": {
+                      "type": "string",
+                      "example": "Partial Content"
```

Comment:
400 can't be Partial Content.
There are many optional descriptions. Choose one of them from the code.